### ISSUE TITLE
Set up web client

### ISSUE THREAD

**Issue by meganniu** [uwblueprint/pregnancy-centre#1]
Submit a separate PR for every task :) It'll be easier to review.

Tasks
+ Under the folder `client/` in the root of this repo:
  + Create a react app that compiles Typescript. 
  + Add tslint
  + Start [here](https://github.com/facebook/create-react-app/releases/tag/v2.1.0).
+ Set up redux state/action/reducer to enable a request query to GraphQL
  + This task is partially blocked by #2 which exposes the GraphQL endpoint
+ Explore options for organizing component styling and components. Present the explorations during a work session so the team can decide together how to organize files.
  + We will be using [bootstrap components](https://getbootstrap.com/2.3.2/components.html)
